run_release.py

```diff
@@ -636,7 +641,7 @@ def sign_source_artifacts(db: ReleaseShelf) -> None:
         subprocess.check_call('gpg -K | grep -A 1 "^sec"', shell=True)
         uid = input("Please enter key ID to use for signing: ")
 
-    tarballs_path = Path(db["git_repo"] / str(db["release"]) / "src")
+    tarballs_path = Path(db["git_repo"] / str(db["release"]) / "downloads")
```

Comment:
*Not really related to this PR* but `db["git_repo"]` is always a `Path`. Otherwise, `db["git_repo"] / str(db["release"])` would explode on strings. And so the entire `db["git_repo"] / str(db["release"]) / "downloads"` is also a `Path`. So wrapping it with `Path()` is rather unexpected:
```suggestion
    tarballs_path = db["git_repo"] / str(db["release"]) / "downloads"
```

*(this was probably copied from `docs_path = Path(db["git_repo"]) / str(release_tag) / "docs"` at some point that only wraps `db["git_repo"]`, but this copy somehow moved the paren; not that it's needed over there either...)*